.github/workflows/build.yml

```diff
@@ -98,3 +98,55 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - "1.18"
+          - "1.19"
+    steps:
+      # We need the Go version and Go cache location for the actions/cache step,
+      # so the Go installation must happen before that.
+      - id: setup-go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Lookup Go cache directory
+        id: go-cache
+        run: |
+          echo "dir=$(go env GOCACHE)" >> $GITHUB_OUTPUT
+      - uses: actions/cache@v3
+        env:
+          BASE_CACHE_KEY: "${{ github.job }}-${{ runner.os }}-\
+            go${{ steps.setup-go.outputs.go-version }}-"
+        with:
+          path: |
+            ${{ steps.go-cache.outputs.dir }}
+          key: "${{ env.BASE_CACHE_KEY }}\
+            ${{ hashFiles('go.mod') }}-\
+            ${{ hashFiles('go.sum') }}"
+          restore-keys: |
+            ${{ env.BASE_CACHE_KEY }}
+      # Cache go build cache, used to speedup go test
+      - name: Go Build Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ steps.go-cache.outputs.dir }}
+          key: "${{ env.BASE_CACHE_KEY }}\
+            ${{ hashFiles('go.mod') }}-\
+            ${{ hashFiles('go.sum') }}"
+
+      # Cache go mod cache, used to speedup builds
+      - name: Go Mod Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{ steps.go-cache.outputs.dir }}
+          key: "${{ env.BASE_CACHE_KEY }}\
+            ${{ hashFiles('go.mod') }}-\
+            ${{ hashFiles('go.sum') }}"
```

Comment:
These are essentially duplicates of the earlier caching step and now there is nothing being done in this job.

Reply:
@mcdonnnj so to clarify the initial caching step builds and tests the go application?

Reply:
resolved with [5b1493b](https://github.com/cisagov/con-pca-tasks/pull/1/commits/5b1493b033102587c6a7655d569ca0ce321f8f8f)